<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pt::GeneralPreferencesPage</class>
 <widget class="QWidget" name="pt::GeneralPreferencesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="userInterfaceGroup">
     <property name="title">
      <string>userInterface</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="languageLabel">
        <property name="text">
         <string>language</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="languages"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="miscGroup">
     <property name="title">
      <string>misc</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0,0" columnstretch="0,0">
      <item row="5" column="0">
       <widget class="QLabel" name="startPositionLabel">
        <property name="text">
         <string>startPosition</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="startPosition"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="enableAutomaticCrashReporting">
        <property name="text">
         <string>enableAutomaticCrashReporting</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="enableGeoIPLookups">
        <property name="text">
         <string>enableGeoIPLookups</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="skipAddTorrentDialog">
        <property name="text">
         <string>skipAddTorrentDialog</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="startWithWindows">
        <property name="text">
         <string>startPicoTorrentWithWindows</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="notificationGroup">
     <property name="title">
      <string>notificationArea</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="showInNotificationArea">
        <property name="text">
         <string>showInNotificationArea</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>15</number>
        </property>
        <item>
         <widget class="QCheckBox" name="minimizeToNotificationArea">
          <property name="text">
           <string>minimizeToNotificationArea</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="closeToNotificationArea">
          <property name="text">
           <string>closeToNotificationArea</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
